packages/react-storage/src/components/StorageBrowser/Views/Controls/Exit.tsx

```diff
@@ -22,16 +23,17 @@ interface ExitProps {
 
 export interface ExitControl<
   T extends StorageBrowserElements = StorageBrowserElements,
-> extends OmitElements<_ExitControl<T>, 'Button'> {
+> extends OmitElements<_ExitControl<T>, 'Button' | 'Icon'> {
   ({ onClick }: ExitProps): React.JSX.Element;
 }
 
 const Button = withBaseElementProps(ButtonElement, {
   className: `${BLOCK_NAME}`,
   variant: 'exit',
-  children: 'Back',
 });
 
 export const ExitControl: ExitControl = ({ onClick }) => (
-  <Button onClick={onClick} />
+  <Button onClick={onClick}>
+    <Icon variant="exit" /> Back
```

Comment:
nit: should we start collecting all the strings in one place so it will be easier to refactor them?